.circleci/config.yml

```diff
@@ -13,7 +13,12 @@ jobs:
       # -batch is necessary to make sure sbt doesn't stall waiting for user input in case of certain failures
       - run: echo '-batch' >> .sbtopts
       - run: sbt update
-      - run: sbt +scalafmtCheckAll
+      - run:
+          name: Checking formating
+          command: sbt "scalafmtCheckAll"
```

Comment:
No `+` anymore? 🤔

Reply:
According to [this documentation](https://www.scala-sbt.org/1.x/docs/Cross-Build.html), a `+` operation is used when cross building using setting `crossScalaVersions`, but cross building is done by `ProjectMatrix`

Reply:
Ok! so we don't need `+` anywhere now AFAICS